### ISSUE TITLE
Search button is not labeled 

### ISSUE THREAD

**Issue by chiefcll** [zalando-incubator/hexo-theme-doc#128]
## Expected Behavior
Using a screen reader, the search button should say "Search"

## Actual Behavior
Read as Button

## Steps to Reproduce the Problem

  1. Go to main site with screen reader, Read search area
  2.
  3.

## Specifications

  - NodeJS Version:
  - NPM Version:
  - OS:
  - Hexo Version:

This can be fixed here: https://github.com/zalando-incubator/hexo-theme-doc/blob/8e51c8a74fce09fea6de7eb0a1942ed2fbceaf9b/lib/browser/search/components.jsx#L41

Two ways to fix.. Add text inside the button as "Search" and then move it off screen doing something like
```
   .screen-reader-text {
        position: absolute !important;
        display: block;
        visibility: visible;
        overflow: hidden;
        width: 1px;
        height: 1px;
        margin: -1px;
        border: 0;
        padding: 0;
        clip: rect(0px, 0px, 0px, 0px); /* deprecated */
        clip-path: polygon(0px 0px, 0px 0px, 0px 0px, 0px 0px);
    }
```
Or add an aria-label to the button with Search

**Comment by mikeyil** [comment]
@rbarilani, I've see you've labeled this as an enhancement. Is it really an enhancement when a screen reader user can't tell what the button does at all?

**Comment by rbarilani** [comment]
@mikeyil It's absolutely a **must-have** but for our "label main types" this issue is categorised as "enhancement" because it's not a new "feature" or a "bug" which are the other two categories.

I added an accessibility flag to make this kind of issues more visible.
